packages/backend/src/config/features/tvl.ts

```diff
@@ -85,9 +85,9 @@ export function getChainTvlConfig(
             },
       minBlockTimestamp:
         options?.minTimestamp ?? chainConfig.minTimestampForTvl,
-      multicallConfig: (chainConfig.multicallContracts ?? []).map(
-        toMulticallConfigEntry,
-      ),
+      multicallConfig: (chainConfig.multicallContracts ?? [])
```

Comment:
If the logic of MulticallClient relies on this sorting - then you have two options: either sort it here and do the assert, or sort in the MulticallClient. I would personally sort it in the client